pkg/plugin/v3/init.go

```diff
@@ -136,6 +151,16 @@ func (p *initPlugin) Validate() error {
 }
 
 func (p *initPlugin) GetScaffolder() (scaffold.Scaffolder, error) {
+	// Add core resource versions to the config if they are not the defaults so we don't unnecessarily
+	// clutter the resulting PROJECT file.
+	if p.crdVersion != defaultCRDVersion {
+		p.config.AddCoreResourceVersions(config.CRDResourceName, []string{p.crdVersion})
+	}
+	if p.webhookVersion != defaultWebhookVersion {
+		p.config.AddCoreResourceVersions(config.MWebhookResourceName, []string{p.webhookVersion})
+		p.config.AddCoreResourceVersions(config.VWebhookResourceName, []string{p.webhookVersion})
+	}
+
```

Comment:
Also, if we : 

$kb create api --group ship --version v1beta1 --kind Frigate --controller=true --resource=true --make=false --apiversion=v1
$kb create webhook --group ship --version v1beta1 --kind Frigate --conversion -apiversion=betav1

Would not it fail?